docs/search.md

```diff
@@ -96,6 +96,16 @@ The search button displays in the bottom right corner of the screen and triggers
 search.button: true
 ```
 
+### Include posts in search
+
+By default the search only indexes pages. Posts can be included in the search index.
+
+```yaml
+# Enable or disable the indexing of Jekyll posts
+# Supports true or false (default)
+search.posts: true
```

Comment:
Any reason why not to instead write

```suggestion
search:
    posts: true
```

both to match consistency / reduce confusion?